docker-compose.yaml

```
@@ -0,0 +1,10 @@
+services:
+  simplemind:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./simplemind:/src/simplemind
+      - ./build.py:/src/build.py
+    env_file:
+      - .env
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)